starlite/app.py

```diff
@@ -540,6 +541,42 @@ def get_membership_details(group_id: int, user_id: int) -> None:
 
         return join_paths(output)
 
+    def asset_url_path(self, name: str, file_path: str) -> str:
```

Comment:
as suggested above- `create_url_for_static_asset` or simply `url_for_static_asset` if you want a somewhat shorter version